apps/eo/e2e-eo/src/e2e/transfers/transfers.feature

```diff
@@ -11,3 +11,19 @@ Feature: Transfers Page
     And I can see a download button
     And I can see a button to create a new agreement
 
+  Scenario: Charlotte CSR can access a modal for creating new transfer agreement on the transfers page
```

Comment:
Question: Aren't this scenario redundant as it's necessary to be able to access the modal to create the transfer agreement. Suggestion from my side would be to remove this scenario and instead add a scenario checking that we handle errors correctly.

Reply:
Removed :)